lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -74,7 +74,7 @@ class ArynPDFPartitioner:
     ArynPartitioner class.
     """
 
-    def __init__(self, model_name_or_path=ARYN_DETR_MODEL, device=None):
+    def __init__(self, model_name_or_path, device=None, cache: Optional[Cache] = None):
```

Comment:
Why drop the default parameter value?  Doesn't seem to have much to do with caching.

Reply:
it was a mistake